README.md

```
@@ -39,14 +39,19 @@ flowchart LR
 
 **Installation**
 
-One-liner (recommended):
+**Docker (recommended)** — bundles SAW + OpenClaw gateway in a single image. SAW is built from source, OpenClaw is installed via npm. See [Docker](#docker) below.
```

markdownlint-cli2 (0.20.0)

[warning] Link fragments should be valid

(MD051, link-fragments)